src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.RenameTrackingOperation.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.RenameTracking
+{
+    internal sealed partial class RenameTrackingTaggerProvider
+    {
+        /// <summary>
+        /// This class exists for test usage without fully exposing the private classes of RenameTrackingTaggerProvider
```

Comment:
What else outside of the rename tracking code is already calling into this? I guess I'm bit a confused why this is here.

Reply:
RenameTrackingTestState uses CodeAction for testing